src/strategies/erc20-balance-from-graph/index.ts

```diff
@@ -0,0 +1,52 @@
+import { subgraphRequest } from '../../utils';
+import { BigNumber } from '@ethersproject/bignumber';
+
+export const author = 'yangz';
+export const version = '1.0.0';
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const query = {
+    accounts: {
+      __args: {
+        where: {
+          id_in: addresses
+        },
+        first: 1000
```

Comment:
Can we make it like pagination to get balance all addresses instead of the first 1000
Something like 
```
const addressSubsets = Array.apply(
    null,
    Array(Math.ceil(addresses.length / LIMIT))
  ).map((_e, i) => addresses.slice(i * LIMIT, (i + 1) * LIMIT));
  const returnedFromSubgraph = await Promise.all(
    addressSubsets.map((subset) =>
      mcnSubgraphQuery(subset, snapshotBlock)
    )
  );
  ```